lib/biran/configurinator.rb

```diff
@@ -121,7 +131,7 @@ def sanitize_config_files files_list
       lambda do |file, _|
         files_list[file] ||=  {extension: ''}
         ext = files_list[file].fetch(:extension, '').strip
-        ext.prepend('.') unless ext.starts_with?('.') || ext.empty?
+        ext.prepend('.') unless ext.start_with?('.') || ext.empty?
```

Comment:
So this is a ruby vs rails change, but don't we get `starts_with?` because of ActiveSupport?

Reply:
yes, so that is why the `activesupport` requirement added... also, is `start_with?` there was a typo in here

Reply:
Its not a typo. It was working. It looks to me more like the rails version is using the ruby version now anyway. But not a typo.

Reply:
Probably, I might need to require another rails dependency then

Reply:
But if there is no difference in the output between the core rails method and Ruby one I prefer the Ruby core one.